src/layouts/DefaultLayout.vue

```diff
@@ -1,7 +1,14 @@
 <script setup>
 import { RouterView } from 'vue-router'
+import AppBar from '../components/AppBar.vue'
```

Comment:
jeśli to możliwe to staraj się używać @ w importach. Na serwerach linuxowych lubią być z tymi ścieżkami problemy. Zależy w sumie od hostingu.